server/chat-commands/admin.ts

```diff
@@ -570,6 +569,7 @@ export const commands: Chat.ChatCommands = {
 			return this.errorReply("Wait for /updateserver to finish before hotpatching.");
 		}
 
+		await this.parse(`/rebuild`);
```

Comment:
In the past, we've rebuilt after `/updateserver` as well because the server was configured to not rebuild on startup in order to save time during restarts. Is `esbuild` fast enough that it won't add much downtime to restarts, or should we continue building in `/updateserver` like we did when we used sucrase?

Reply:
The current build script takes 4 seconds to run. Up to you.

Reply:
Let's leave it like this for now. 4 seconds on your computer is probably 1-3 seconds on sim3, which may not be noticeable. If it is bothering people we should change it back.

Reply:
OK.